### PR TITLE
Changes to make count param in historian query to apply to each individual topic queried

### PR DESCRIPTION
**Pull request by schandrika**
This change is [Reviewable](https://reviewable.io/reviews/volttron/volttron/850)

### CODE REVIEW

File: services/core/MongodbHistorian/mongodb/historian.py [hunk 1]
```diff
@@ -89,13 +89,21 @@ def historian(config_path, **kwargs):
...
-                     " See scripts/core/make-mongo-historian.sh for an example of how this is done.")
+                     " from its configuration file will no longer be "
+                     "supported after VOLTTRON 4.0")
+        _log.warning(
+            "DEPRECATION WARNING: Using the identity configuration setting "
+            "will override"
+            " the value provided by the platform. This new value will not be "
+            "reported"
+            " correctly by 'volttron-ctl status'")
+        _log.warning(
```
Comment (callwardt) [inline services/core/MongodbHistorian/mongodb/historian.py]: No this format of strong isn't acceptable

File: services/core/MongodbHistorian/tests/test_mongohistorian.py [hunk 2]
```diff
@@ -661,3 +661,66 @@ def test_empty_result(volttron_instance, database_client):
...
+            order="FIRST_TO_LAST").get(timeout=100)
+
+        # print("expected result {}".format(expected_result))
+        # print("result {}".format(result))
+        assert result["metadata"] == expected_result["metadata"]
+        assert result["values"][query_points['mixed_point']] == \
+               expected_result["values"][query_points['mixed_point']]
+        assert result["values"][query_points['oat_point']] == \
+               expected_result["values"][query_points['oat_point']]
+
```
Comment (callwardt) [inline services/core/MongodbHistorian/tests/test_mongohistorian.py]: Too many lines here

File: services/core/SQLHistorian/sqlhistorian/historian.py [hunk 3]
```diff
@@ -85,13 +85,21 @@ def historian(config_path, **kwargs):
...
-                     " correctly by 'volttron-ctl status'")
-        _log.warning("DEPRECATION WARNING: Please remove 'identity' from your configuration file"
-                     " and use the new method provided by the platform to set an agent's identity."
-                     " See scripts/core/make-sqlite-historian.sh for an example of how this is done.")
+                     " from its configuration file will no longer be "
+                     "supported after VOLTTRON 4.0")
+        _log.warning(
+            "DEPRECATION WARNING: Using the identity configuration setting "
+            "will override"
+            " the value provided by the platform. This new value will not be "
```
Comment (callwardt) [inline services/core/SQLHistorian/sqlhistorian/historian.py]: Not acceptable line formatting

File: services/core/SQLHistorian/tests/test_sqlhistorian.py [hunk 4]
```diff
@@ -292,11 +293,11 @@ def sqlhistorian(request, volttron_instance, query_agent):
         meta_table = 'meta'
     elif request.param['tables_def']['table_prefix']:
         data_table = request.param['tables_def']['table_prefix'] + "_" + \
```
Comment (callwardt) [inline services/core/SQLHistorian/tests/test_sqlhistorian.py]: These changes are not nessesary and make it harder to read

**Comment by schandrika** [comment]
fulled changes from develop, reformatted and pushed changes.

**Comment by schandrika** [comment]
Review status: 0 of 7 files reviewed at latest revision, 4 unresolved discussions.

---

*[services/core/MongodbHistorian/mongodb/historian.py, line 100 at r1](https://reviewable.io:443/reviews/volttron/volttron/850#-KTo_UDsdAX-1KgMXsc1-r1-100:-KTuTt2C6m_4JSz1rCj7:b-896fix) ([raw file](https://github.com/volttron/volttron/blob/65bf3b5316480dd0bc889d1a52386960c6343889/services/core/MongodbHistorian/mongodb/historian.py#L100)):*
<details><summary><i>Previously, callwardt wrote…</i></summary>
> The layout change of this seems that it could be better.
</details>
Done.

---

*[services/core/MongodbHistorian/tests/test_mongohistorian.py, line 721 at r1](https://reviewable.io:443/reviews/volttron/volttron/850#-KTo_UDsdAX-1KgMXsc2-r1-721:-KTuTutJvLT5nwLhEiee:b-896fix) ([raw file](https://github.com/volttron/volttron/blob/65bf3b5316480dd0bc889d1a52386960c6343889/services/core/MongodbHistorian/tests/test_mongohistorian.py#L721)):*
<details><summary><i>Previously, callwardt wrote…</i></summary>
> Too many lines here
</details>
Done.

---

*[services/core/SQLHistorian/sqlhistorian/historian.py, line 93 at r1](https://reviewable.io:443/reviews/volttron/volttron/850#-KTo_UDsdAX-1KgMXsc3-r1-93:-KTuTvJFVGg3GcRWztEB:b-896fix) ([raw file](https://github.com/volttron/volttron/blob/65bf3b5316480dd0bc889d1a52386960c6343889/services/core/SQLHistorian/sqlhistorian/historian.py#L93)):*
<details><summary><i>Previously, callwardt wrote…</i></summary>
> Not acceptable line formatting
</details>
Done.

---

*[services/core/SQLHistorian/tests/test_sqlhistorian.py, line 295 at r1](https://reviewable.io:443/reviews/volttron/volttron/850#-KTo_UDsdAX-1KgMXsc4-r1-295:-KTuTvsMdK0jnp3nHDYL:b-896fix) ([raw file](https://github.com/volttron/volttron/blob/65bf3b5316480dd0bc889d1a52386960c6343889/services/core/SQLHistorian/tests/test_sqlhistorian.py#L295)):*
<details><summary><i>Previously, callwardt wrote…</i></summary>
> These changes are not nessesary and make it harder to read
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/volttron/volttron/850)*

**Pull request merged**